internal/jwt/container_test.go

```diff
@@ -0,0 +1,129 @@
+package jwt
+
+import (
+	"errors"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestCookieContainer(t *testing.T) {
+	req := httptest.NewRequest("GET", "/", nil)
+	cl := Claims{}
+	cl.Subject = "jhon.doe"
```

Comment:
Seems like this can be condensed into a common test setup function.  Its duplicated in a couple of places.